en/frames_multicopter/omnicopter.md

```diff
@@ -0,0 +1,107 @@
+- TODO: 3D parts + screws + standoffs + rod lengths
+
+![Parts List](../../assets/airframes/multicopter/omnicopter/parts_list.jpg)
+
+## Assembly
+
+### Frame
+- Print the 3D parts
+- Cut the rods
+- Test that it all works by connecting the frame pieces together:
+![Frame](../../assets/airframes/multicopter/omnicopter/frame_only.jpg)
+- Place the motors as far out as possible, without the propellers touching the rods.
+
+### Electronics
+Solder the peripherals to the flight controller. We used the following assignments:
+- ESCs: the 2 ESCs can be connected directly to the two connectors of the KakuteH7. To avoid conflicts we removed the power pin (right-most pin) from one of the connectors.
+- Telemetry to UART1
+- GPS to UART4
+- RC to UART6
+![FC closeup](../../assets/airframes/multicopter/omnicopter/fc_closeup.jpg)
+
+Remarks:
+- Make sure the magnetometer is placed away from power. We ended up placing it to the bottom of the center-piece with a 4cm padding styrofoam.
+- Put some tape on the barometer (without taping the opening!) to avoid any influence from light.
+- We did not glue the frame. It is certainly advisible to do so after initial test flights, but it might work without.
+
+
+## Software Configuration
+
+### ESC
+First, configure the ESCs to 3D mode (bidirectional).
+We had issues with the stock ESC settings in 3D mode: when trying to switch direction, a motor sometimes did not start anymore until the ESC was rebooted.
+So we had to change ESC settings.
+To do so, you can use Betaflight on the flight controller and then use pass-through mode and the BL Heli suite (make sure there is an airframe with 8 motors configured in Betaflight).
+These are the settings:
+![ESC settings](../../assets/airframes/multicopter/omnicopter/esc_settings.png)
+In particular:
+  - set the Motor Direction to **Bidirectional Soft**
+  - increase the Rampup Power to **100%** (this is conservative and might reduce efficiency)
+
+:::note
+Make sure the motors do not overheat with the changed settings.
+:::
+
+### PX4
+- Use an arming switch, do not use stick arming
+- Select DShot as output protocol
+- Configure the motors according to this:
+![Motor Config](../../assets/airframes/multicopter/omnicopter/motors_configuration.png)
+  We used the following convention: the motors are facing the direction of where the axis points to. The spinning direction matches the direction of positive thrust.
+  Make sure to use the right props, as there is a CCW and a CW version.
+- Parameters:
+  - Change the desaturation logic for better attitude tracking: set [CA_METHOD](../advanced_config/parameter_reference.md#CA_METHOD) to 0.
+  - Disable failure detection: set [FD_FAIL_P](../advanced_config/parameter_reference.md#FD_FAIL_P) and [FD_FAIL_R](../advanced_config/parameter_reference.md#FD_FAIL_R) to 0.
+
```

Comment:
I would make a remark about the tuning. Need to re-check the vehicle, but I think we only had to slightly reduce the yawrate P for rate, and higher level I'm not sure anymore.